Documentation/ColumnsConfig/Type/Inline/Examples.rst

```diff
@@ -136,3 +136,97 @@ added to the parent record.
 
 .. include:: /CodeSnippets/InlineUsecombinationcInline1.rst.txt
 
+.. _inline-example-field-information:
+
+Add a custom fieldInformation
+=============================
+
+We show a very minimal example which adds a custom fieldInformation for the
+inline type in tt_content. Adding a fieldWizard is done in a similar way.
+
+As explained in the :ref:`description <columns-inline>`, :code:`fieldInformation`
+or :code:`fieldWizard` must be configured within the :code:`ctrl` **for the field
+type inline** - as it is a container.
+
+.. rst-class:: bignums-xxl
+
+#. Create a custom fieldInformation
+
+   .. code-block:: php
+      :caption: EXT:myext/Classes/FormEngine/FieldInformation/DemoFieldInformation
```

Comment:
We usually use "my_extension" and "my_sitepackage" to stay consistent
```suggestion
      :caption: EXT:my_extension/Classes/FormEngine/FieldInformation/DemoFieldInformation
```

Reply:
Ok, the naming conventions recommend to avoid underscores in extension keys - so we have competing "standards" here (lol). I'll change it.